chaosazure/machine/actions.py

```diff
@@ -217,11 +238,11 @@ def __fetch_machines(filter, configuration, secrets):
     return machines
 
 
-def __fetch_machine_at_random(filter, configuration, secrets):
+def __fetch_machines_at_random(filter, count, configuration, secrets) -> []:
     machines = __fetch_machines(
         filter, configuration=configuration, secrets=secrets)
-    choice = random.choice(machines)
-    return choice
+    result = random.sample(machines, count)
```

Comment:
Theoretically we can achieve similar results using filter.  So this feture may be redundant 
https://docs.microsoft.com/en-us/azure/kusto/query/takeoperator

Reply:
or 
https://docs.microsoft.com/en-us/azure/kusto/query/sampleoperator

Reply:
this one looks promising

Reply:
Will build this in. Thanks for the hint.